.evergreen/tox.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -o errexit  # Exit the script with error if any of the commands fail
+
+PYTHON_BINARY="$1"
+
+if $PYTHON_BINARY -m tox --version; then
+    alias run_tox='$PYTHON_BINARY -m tox'
+elif python3 -m tox --version; then
+    alias run_tox='python3 -m tox'
+elif tox --version; then
+    alias run_tox=tox
+else # No toolchain present, set up venv before installing tox
```

Comment:
Can we use createvirtualenv from utils.sh?

Reply:
Currently createvirtualenv uses this flow to determine its venv executable:

```
if $PYTHON -m virtualenv --version; then
        VIRTUALENV="$PYTHON -m virtualenv"
elif $PYTHON -m venv -h > /dev/null; then
        # System virtualenv might not be compatible with the python3 on our path
        VIRTUALENV="$PYTHON -m venv"
```

can we remove the first check to migrate off of virtualenv, which is deprecated since Python 3.6?